### PR TITLE
Fix missed parameter from AUC config

### PR DESCRIPTION
**Pull request by weipeilun**
AUC metrics missed `from_logits` parameter in `get_config` method, which will cause a error when load from a saved model and `from_logits` is True.

### CODE REVIEW

**Review by qlzh727 — CHANGES_REQUESTED** [review]
Thanks for the fix. Could u add a unit test to cover this case.

**Review by fchollet — COMMENTED** [review]
Thanks for the PR! Could you add a unit test (saving and loading the metric and checking that the behavior is the same)?

**Comment by weipeilun** [comment]
@qlzh727 @fchollet Thanks for the comment. A unit test is added and another parameter missed is found, thanks again.

File: keras/metrics/metrics_test.py [hunk 1]
```diff
@@ -1755,6 +1755,37 @@ def test_axis(self):
     self.assertAllClose(self.evaluate(result), 1.176, atol=1e-3)
 
 
+@test_combinations.generate(test_combinations.combine(mode=['graph', 'eager']))
+class AUCTest(tf.test.TestCase):
```
Comment (qlzh727) [inline keras/metrics/metrics_test.py:1759]: You can move this to the confusion_matrix_test.py, which has a AUCTest class.

**Review by qlzh727 — CHANGES_REQUESTED** [review]
Thanks for the change.

**Comment by weipeilun** [comment]
@qlzh727 I moved the test case to confusion_matrix_test.py, again found a error caused by dtype mismatch of weights and labels when calling update_state(). Please review that too, thanks.

**Review by qlzh727 — APPROVED** [review]
Thanks for the PR and sorry for the wait.
